plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/config/Settings.java

```diff
@@ -11,6 +12,8 @@ public class Settings {
 
     public static final String MAVEN_REWRITE_PLUGIN_VERSION = "5.34.1";
 
+    public static final InputStream RECIPE_DATA_YAML = Settings.class.getClassLoader().getResourceAsStream("recipe_data.yaml");
```

Comment:
I think it's dangerous to store the InputStream here. I think it's safer to only store the string.

It's public, so you don't know if caller will do with it (close it ?)